src/Library/Lod/LodReader.h

```diff
@@ -30,9 +30,20 @@ class LodReader final {
     bool exists(const std::string &filename) const;
     Blob read(const std::string &filename);
 
+    const auto begin() const {
```

Comment:
I'd advise against exposing `begin`/`end` from a non-container class, a better way would be to just add a `const container &files() const` method. Also this will make the code at the call site more readable.